var/spack/repos/builtin/packages/py-numpydoc/package.py

```diff
@@ -0,0 +1,38 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class PyNumpydoc(PythonPackage):
+    """numpydoc - Numpy's Sphinx extensions"""
+
+    homepage = "https://github.com/numpy/numpydoc"
+    url      = "https://pypi.io/packages/source/n/numpydoc/numpydoc-0.6.0.tar.gz"
+
+    version('0.6.0', '5f1763c44e613850d56ba1b1cf1cb146')
+
+    depends_on('py-setuptools',    type='build')
+    depends_on('py-sphinx@1.0.1:', type='build')
+    depends_on('py-jinja2@2.3:',   type='build')
```

Comment:
From which `setup.py`? I'm not seeing that in the tarball from this PR. All I see is:
```python
requires=["sphinx (>= 1.0.1)"],
```
Maybe you're looking at a different version?